src/test/kotlin/com/kroffle/knitting/controller/design/DesignRouterTest.kt

```diff
@@ -94,13 +98,35 @@ class DesignRouterTest {
 
     @Test
     fun `design 이 잘 생성되어야 함`() {
-        webClient
+        val body = objectMapper.writeValueAsString(design)
+        given(repo.createDesign(any())).willReturn(Mono.just(design))
+        val response = webClient
             .post()
             .uri("/designs/")
             .accept(MediaType.APPLICATION_JSON)
+            .bodyValue(body)
             .exchange()
             .expectStatus()
             .isOk
-            .expectBody<Map<String, String>>()
+            .expectBody(Design::class.java)
+            .returnResult()
+            .responseBody
```

Comment:
```suggestion
            .responseBody!!
```
not null로 지정하고, 아래서 `?` 빼고 쓰는 건 어떠신가요?

Reply:
https://github.com/k-roffle/knitting-service/pull/28/commits/6b023b5d9b25ad05c2b2321411570c86005e9050

반영했습니다!